kolibri_explore_plugin/assets/src/views/CustomChannelPresentationApp.vue

```diff
@@ -31,11 +31,12 @@
       rooturl() {
         const app = getAppNameByID(this.channel.id);
         const url = urls['kolibri:kolibri_explore_plugin:app_custom_presentation']({ app: app });
+        const endlessApp = isEndlessApp(this.channel) ? 'endless=true' : '';
         if (this.customAppContent.id) {
           const { kind, id } = this.customAppContent;
-          return `${url}?${kind}Id=${id}`;
+          return `${url}?${kind}Id=${id}&${endlessApp}`;
```

Comment:
This is sending a new parameter to the custom channel presentation, which wasn't discussed in Learning Equality upstream API (the customApps.js file). The other option is to do it as a template override setting, in the `options.json` of each override. For instance in packages/template-ui/channel-overrides/healthy-living/options.json:

```
{
  "displayLogoInHeader": false,
  "isEndlessApp": true
}
```